### DISCUSSION TITLE
Very good deobfuscator!

### DISCUSSION THREAD

**Discussion by Taureon** [j4k0xb/webcrack#19] [General]
For a decent while, I've been sitting at either making my own deobfuscators or using synchrony.
Then a friend of mine gave me this thing, and it actually blew me away, not gonna lie.
Something that took synchrony multiple seconds without even completely finishing got absolutely destroyed by webcrack in less than a second!
If anything deserves a star, it's this project :)

**Comment by j4k0xb** [comment]
thanks 😉 
synchrony was the best for a quite a while afaik
I initially made this project because of a large file that took like 30 minutes to deobfuscate and ended up with random strings, as well as to improve the output code readability (more unminifying, removing debug protection etc)
theoretically the performance could be improved even more but I wanna first focus on fixing some edge cases with control flow flattening and dead code injection
